### ISSUE TITLE
🚀 [2025/12/8] Recent Updates Summary for ROLL Project

### ISSUE THREAD

**Issue by PanAndy** [alibaba/ROLL#286]
大家好！感谢大家对ROLL的关注。  
ROLL近期更新了大量新功能，以下是近期更新的一些梳理，我们将持续对ROLL进行迭代更新，欢迎加入ROLL的社区。

🚀亮点: 

+ (feat): support Qwen3VL, mcore_adapter and examples.
+ (feat): Add optimization for computing ref_logprobs and old_logprobs.
+ (feat): support vllm beam_search.
+ (feat): Add support for Qwen-3-next on AMD GPUs.
+ (feat): support sglang==0.5.4、vllm==0.11.1、torch2.8.0.

🚀主要新特性：

+ Agentic
    - (fix): fix agentic val get_batch state in redundancy env.
    - (feat): agentic-spec actor worker.
    - (feat): add infer_log_probs in agentic.
    - (feat): refactor agentic norm like LitePPO.
    - (feat): add agentic profile metrics.
+ 模型与后端
    - (feat): support vllm beam_search.
    - (feat): Add support for Qwen-3-next on AMD GPUs.
    - (feat): support offload nccl to save gpu memory. Thanks for slime.
    - (feat): support sglang 054.
    - (feat): sglang support dp-attention.
    - (feat): add enable_reference option. #250 
    - (feat): add enable_old_logprobs, opt old log probs by cache.
    - (feat): support Qwen3VL, mcore_adapter and examples yaml. #190 
    - (feat): add sequence packing for sft pipeline and distill pipeline, optimize memory usage during top-k logits computation.
+ bug fix, refactor
    - (fix): update math rule reward worker with thinking. #281 
    - (feat): set RAY_CGRAPH_get_timeout=600.
    - (fix): fix train infer ratio/diff mean & add train infer ratio/diff token/seq mask & add rollout importance sampling. #242 #273 
    - (fix): ensure compatibility with transformers version check for causal mask update.
    - (fix): fix vllm 0110 import for torch280.
    - (fix): fix tokenizer mismatch between policy and reward model in llm judge reward worker. #91 
    - (fix): fix bugs in data fetching for face embeddings for wan_module.
    - (fix): vllm _generate_standard missing prompt_token_ids input args in vllm >0.11.0. #189 
    - (fix): vllm add missing argument is_lora in function update_parameter. #233 
    - (fix): fix bugs with metrics recording in the DPO pipeline.
    - (fix): update image loading logic for byte data in rlvr_vlm_pipeline.py
    - (fix): add alive check. #253

**Comment by PanAndy** [comment]
Hello everyone! Thanks for your interest in ROLL.  
ROLL has recently introduced a large number of new features. Below is a summary of recent updates. We will continue to iterate on ROLL, and you are very welcome to join the ROLL community.

🚀 Highlights:

+ (feat): Support Qwen3VL, mcore_adapter, and examples.
+ (feat): Add optimization for computing ref_logprobs and old_logprobs.
+ (feat): Support vllm beam_search.
+ (feat): Add support for Qwen-3-next on AMD GPUs.
+ (feat): Support sglang==0.5.4, vllm==0.11.1, torch==2.8.0.

🚀 Main new features:

+ Agentic
    - (fix): Fix agentic val get_batch state in redundancy environments.
    - (feat): Agentic-spec actor worker.
    - (feat): Add infer_log_probs in agentic.
    - (feat): Refactor agentic norm similar to LitePPO.
    - (feat): Add agentic profile metrics.

+ Models & backends
    - (feat): Support vllm beam_search.
    - (feat): Add support for Qwen-3-next on AMD GPUs.
    - (feat): Support offloading NCCL to save GPU memory (thanks to slime).
    - (feat): Support sglang 0.5.4.
    - (feat): Add dp-attention support in sglang.
    - (feat): Add enable_reference option. #250  
    - (feat): Add enable_old_logprobs and optimize old log probs via cache.
    - (feat): Support Qwen3VL, mcore_adapter, and example YAMLs. #190  
    - (feat): Add sequence packing for SFT and distillation pipelines, and optimize memory usage during top-k logits computation.

+ Bug fixes & refactors
    - (fix): Update math rule reward worker with thinking. #281  
    - (feat): Set RAY_CGRAPH_get_timeout=600.
    - (fix): Fix train/infer ratio & diff mean; add train/infer ratio/diff token/seq mask; add rollout importance sampling. #242  
    - (fix): Ensure compatibility with transformers’ version check for causal mask update.
    - (fix): Fix vllm 0.11.0 import for torch 2.8.0.
    - (fix): Fix tokenizer mismatch between policy and reward model in llm judge reward worker. #91  
    - (fix): Fix bugs in data fetching for face embeddings in wan_module.
    - (fix): Fix missing prompt_token_ids argument in vllm._generate_standard for vllm > 0.11.0.
    - (fix): Add missing is_lora argument in vllm.update_parameter. #233  
    - (fix): Fix bugs with metrics recording in the DPO pipeline.
    - (fix): Update image loading logic for byte data in rlvr_vlm_pipeline.py.
    - (fix): Add alive check. #253

**Comment by PanAndy** [comment]
release codes: #283
